src/views/KeyResultHome.vue

```diff
@@ -280,23 +280,25 @@ export default {
         await this.$bind('progress', db.collection(`keyResults/${keyResult.id}/progress`).orderBy('timestamp', 'desc'));
         this.isLoading = false;
         this.value = this.progressDetails.totalCompletedTasks;
-        this.graph = new LineChart(this.$refs.graph, { colorMode: this.theme });
+        this.graph = new LineChart(this.$refs.graph);
```

Comment:
Vi trenger ikke spesifisere `theme` her (som i `mounted`)? For så vidt ikke relatert til denne PRen, men kunne kanskje trekt ut duplisert `this.graph = new LineChart(...)` og `this.graph.render(...)` i en egen metode?

Reply:
Fikset det første. Det andre tror jeg hører hjemme i en egen PR, siden denne begynner å bli ganske stor. :sweat_smile: